.drone.yml

```diff
@@ -4,40 +4,41 @@ workspace:
 
 services:
   web:
-    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:7.3-ci}
+    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:8.0-ci}
     environment:
       - DOCUMENT_ROOT=/test/oe-poetry-client
 
 pipeline:
   composer-install:
     group: prepare
-    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:7.3-ci}
+    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:8.0-ci}
     volumes:
       - /cache:/cache
     commands:
       - composer update ${COMPOSER_BOUNDARY}
 
   grumphp:
     group: test
-    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:7.3-ci}
+    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:8.0-ci}
     commands:
       - ./vendor/bin/grumphp run
 
   phpunit:
     group: test
-    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:7.3-ci}
+    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:8.0-ci}
     commands:
       - ./vendor/bin/phpunit
 
   phpspec:
     group: test
-    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:7.3-ci}
+    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:8.0-ci}
     commands:
       - ./vendor/bin/phpspec run
 
 matrix:
   IMAGE_PHP:
-    - registry.fpfis.eu/fpfis/httpd-php:7.3-ci
+    - registry.fpfis.eu/fpfis/httpd-php:8.0-ci
+    - registry.fpfis.eu/fpfis/httpd-php:7.4-ci
```

Comment:
Why do we need a 7.4 matrix if we don't use that?

Reply:
We are using it for tests, just like in other component on drone. See: https://drone.fpfis.eu/ec-europa/oe-poetry-client/89